test/core.jl

```diff
@@ -5089,3 +5089,23 @@ m22929_2.x = m22929_1
 @test !isdefined_22929_x(m22929_1)
 @test isdefined_22929_1(m22929_2)
 @test isdefined_22929_x(m22929_2)
+
+# issue 18933
+module GlobalDef18933
+    using Base.Test
+    f() = (global sin; nothing)
```

Comment:
Sure. That operation gives a warning now, so presumably it would be less breaking for it to be ignored (have `global x` mean more consistently the statement "x is a global" rather than the being sometimes the command "make x global").